install/restart_enterprise.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# cleaning the database 
+source variables.sh
+
+
+PUBLIC_SERVER_IP="$(jq -r '.HEADNODE_IP | .[0]' config.json)"
+PRIVATE_SERVER_IP="$(jq -r '.PRIVATE_HEADNODE_IP | .[0]' config.json)"
+
+PASSWORD=$db_password
+USERNAME=$admin_name
+PG_CONN_STRING="postgresql://modelbazaaruser:$PASSWORD@$PRIVATE_SERVER_IP:5432/modelbazaar"
+
+
+ssh "$USERNAME"@$PUBLIC_SERVER_IP <<EOF
+if ! command -v psql &> /dev/null; then
```

Comment:
Instead of using sql commands to delete the sql data, can we just delete the sql files in the directory $shared_dir/database/data? And then we can restart the docker container that is running the postgresql database. 

This would ensure a clean start of the db for all aspects. For example, I think right now the schema of the db is not reset with the current code, but the above method would reset that (this is useful if the schema somehow is in a broken state).

Reply:
> I think right now the schema of the db is not reset with the current code
yes, the current implementation just truncate the table